infra/gravity/status.go

```diff
@@ -17,30 +17,141 @@ limitations under the License.
 package gravity
 
 import (
+	"bufio"
 	"context"
+	"encoding/json"
 
 	"github.com/cenkalti/backoff"
 	"golang.org/x/sync/errgroup"
 
-	"github.com/gravitational/robotest/lib/constants"
 	sshutils "github.com/gravitational/robotest/lib/ssh"
 	"github.com/gravitational/robotest/lib/utils"
 	"github.com/gravitational/robotest/lib/wait"
 	"github.com/gravitational/trace"
 )
 
+// ClusterState* consts come from https://github.com/gravitational/gravity/blob/7.0.0/lib/ops/constants.go#L64-L93
+const (
+	// ClusterStateActive is healthy and not running any operations.
+	ClusterStateActive = "active"
+	// ClusterStateDegraded is unhealthy.
+	ClusterStateDegraded = "degraded"
+)
+
+// SystemStatus comes from https://github.com/gravitational/satellite/blob/7.1.0/agent/proto/agentpb/agent.pb.go#L28-L32
+type SystemStatus int
+
+const (
+	SystemStatus_Unknown  SystemStatus = 0
```

Comment:
Nit: All top-level exported names should have Godocs on them.

Reply:
Documented in 3051902.